solidus_shipstation.gemspec

```diff
@@ -31,4 +31,6 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'sass-rails'
   s.add_development_dependency 'sqlite3'
   s.add_development_dependency 'rspec-xsd'
+  s.add_development_dependency 'pry'
+  s.add_development_dependency 'pry-byebug'
```

Comment:
That's what people use until they find [Irbtools|https://github.com/janlelis/irbtools]

![screenshot from 2016-04-28 20 58 02](https://cloud.githubusercontent.com/assets/1651750/14899455/f2b8e708-0d83-11e6-97b1-ced0f3195e7d.png)

And [LogBuddy](https://github.com/relevance/log_buddy/) -> outputs to your own log - `log/awesome.log`, and you just `d {anything}` and you get your stuff in colour and style like this:


--
![screenshot from 2016-04-28 21 00 54](https://cloud.githubusercontent.com/assets/1651750/14899522/556d0cb2-0d84-11e6-861c-83539f98be0c.png)


: )